hunting-buddy.lic

```diff
@@ -252,11 +265,39 @@ class HuntingBuddy
     stop_on_low_skills && stop_on_low_skills.flatten.any? { |skill| DRSkill.getxp(skill) < @stop_on_low_threshold }
   end
 
+  # Is your current encumbrance at or higher than the threshold?
+  # Threshold is a number from 0 (None) to 11 (It's amazing you aren't squashed!)
+  def encumbered?
+    return false unless @hunting_encumbrance_threshold
+    fput('encumbrance') # drinfomon script will parse the output then update DRStats
```

Comment:
Should there be a function on DRStats to refresh your encumbrance, that this just calls?

Reply:
Yep, I'll move that logic to drinfomon and silence the output so that DRStats can always return the current encumbrance value without this hack

Reply:
I agree with this feedback and will follow up on how to do it. For the initial version of this change, I'd like to roll it out with the `encumbrance` command being called once a minute (current design). Amid all the other combat scroll, I don't really notice it myself. Thanks